--- origsrc/gnome-characters-3.18.1/lib/Makefile.am	2015-07-20 16:36:36.000000000 -0500
+++ src/gnome-characters-3.18.1/lib/Makefile.am	2015-11-13 11:16:44.632363700 -0600
@@ -13,7 +13,7 @@ AM_CPPFLAGS =					\
 pkglib_LTLIBRARIES = libgc.la
 
 libgc_la_LIBADD = $(LIBGC_LIBS) $(LTLIBICONV) $(LTLIBUNISTRING) $(PANGOFT2_LIBS) $(builddir)/../gllib/libgnu.la
-libgc_la_LDFLAGS = -avoid-version
+libgc_la_LDFLAGS = -module -avoid-version -no-undefined
 libgc_la_sources = gc.h gc.c
 libgc_la_SOURCES = $(libgc_la_sources) confusables.h scripts.h
 
